index.html

```diff
@@ -11,13 +11,19 @@
             <div>
                 <ul class="container nav-main">
                     <li class="nav-buttons">
-                        <a href="#contact">contact</a>
+                        <a href="#contact">
+                        <img src="nav-bar-icon-contact.png" alt="James Cullen Contact">
+                        </a>
                     </li>
                     <li class="nav-buttons">
-                        <a href="#portfolio">portfolio</a>
+                        <a href="#portfolio">
+                        <img src="nav-bar-icon-portfolio.png" alt="James Cullen Portfolio">
+                        </a>
                     </li>
                     <li class="nav-buttons">
-                        <a href="#about">about</a>
+                        <a href="#about">
+                        <img src="nav-bar-icon-about-me-link.png" alt="James Cullen About Me">
```

Comment:
IMG tag needs to be indented, also on line 15 and 20

Reply:
done